recipes/bgen/bld.bat

```diff
@@ -0,0 +1,15 @@
+mkdir build
+if errorlevel 1 exit 1
+cd build
+if errorlevel 1 exit 1
```

Comment:
You don't really need the `if errorlevel 1 exit 1` for creating and entering the directory.
If this fails, which is unlikely, the cake below will fail. (It does not hurt to leave it there though.)